x/cosmwasm/bindings/types.go

```diff
@@ -0,0 +1,32 @@
+package cosmwasm
+
+type Swap struct {
+	PoolId   int    `json:"pool_id"`
```

Comment:
PoolId should be uint64 everywhere.

That matches what [they use in osmosis protobuf](https://github.com/confio/osmosis/blob/main/x/gamm/types/query.pb.go#L485-L487) so we can directly move this field into a Protobuf message. (We also the that type in rust, following the protobuf definition)